### PR TITLE
Removes costume landmarks, replaces with spawners

### PR DESCRIPTION
**Pull request by coiax**
Every costume landmark is now replaced with a lootdrop spawner populated
with all costume `spawner bundles`. Bundles are like lootdrops, except
they spawn 1 of every item, rather than X from a random list.

Also added a minor category to the lootdrops, for these fucking "50%
chance kitty ears, 50% chance rabbit ears" costumes, rather than making
the code overly complex.

- Makes lootdrop use Initialize

No user facing changes.

### CODE REVIEW

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -6,16 +6,19 @@
 	var/lootdoubles = 1		//if the same item can be spawned twice
 	var/list/loot			//a list of possible items to spawn e.g. list(/obj/item, /obj/structure, /obj/effect)
 
-/obj/effect/spawner/lootdrop/New()
+/obj/effect/spawner/lootdrop/Initialize(mapload)
+	..()
+
 	if(loot && loot.len)
+		var/turf/T = get_turf(src)
 		for(var/i = lootcount, i > 0, i--)
```
Comment (lzimann) [inline code/game/objects/effects/spawners/lootdrop.dm]: `var/i in lootcount to 1 step -1`, please

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 2]
```diff
@@ -6,16 +6,19 @@
...
 	var/list/loot			//a list of possible items to spawn e.g. list(/obj/item, /obj/structure, /obj/effect)
 
-/obj/effect/spawner/lootdrop/New()
+/obj/effect/spawner/lootdrop/Initialize(mapload)
+	..()
+
 	if(loot && loot.len)
+		var/turf/T = get_turf(src)
 		for(var/i = lootcount, i > 0, i--)
 			if(!loot.len) break
```
Comment (lzimann) [inline code/game/objects/effects/spawners/lootdrop.dm]: Can you change this so the break is in a new line?

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 3]
```diff
@@ -194,3 +197,44 @@
...
+	name = "pirate hat or bandana spawner"
+	loot = list(
+		/obj/item/clothing/head/pirate = 1,
+		/obj/item/clothing/head/bandana = 1)
+
+/obj/effect/spawner/lootdrop/minor/twentyfive_percent_cyborg_mask
+	name = "25% cyborg mask spawner"
+	loot = list(
+		/obj/item/clothing/mask/gas/cyborg = 25,
+		"" = 75)
```
Comment (coiax) [inline code/game/objects/effects/spawners/lootdrop.dm]: Empty string means no spawn.

Comment (coiax) [inline code/game/objects/effects/spawners/lootdrop.dm]: So in this case, there's a 75% chance of this lootdrop dropping nothing.

**Pull request merged**